cranelift/zkasm_data/i64_mul.wat

```diff
@@ -0,0 +1,15 @@
+(module
+ (import "env" "assert_eq" (func $assert_eq (param i64) (param i64)))
+ (func $main
+	i64.const 214748364
+	i64.const 107374183
+    i64.const 2
+	i64.mul
+    i64.mul
+    i64.const 107374182
+    i64.const 214748366
+    i64.const 2
+	i64.mul
+    i64.mul
```

Comment:
Hm, I worry that might be comparing two numbers that have simply wrapped around together to the same value. i.e. asserting that the multiplication result is `46116860270173224 % 2**32`, rather than `46116860270173224` in full.)

We better find how to represent `i64.const` proper in zkasm, sounds like a good question to ask polygon folks next week. In the mean time I think it is fine to merge this and perhaps file an issue for a TODO here?

Reply:
Added this to the agenda of upcoming meeting. Also created an [issue](https://github.com/near/wasmtime/issues/53) to remind fix this after we get the answer